Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "Core/Config/MainSettings.h"
 
+#include <chrono>
```

Comment:
It really bugs me that I can't write any chrono that is nicer than all those multiplies but even with MSVC's C++20, it's still quite verbose.

Anyway, I think you can remove this.

Reply:
Ah, no, I was trying to implement it using std::chrono to do the date differences but I couldn't get it to work; that's a leftover from my attempt.

Reply:
`chrono` should be possible but it's a lot more verbose, even with C++20  :(